src/app/migrations/0002_auto_20190723_0726.py

```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8 -*-
```

Comment:
Just  anote - we've run into merge conflicts in the past from committing the migration scripts.  @rtgdk and @techytushar are more experienced Django developers, so I'll defer the process question to you if we should be committing these or running migrate.

Reply:
I have removed the migrations.